k8s/linux-arm64/Dockerfile

```diff
@@ -38,11 +39,23 @@ RUN curl --silent -L "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip"
     ./aws/install && \
     rm -f awscliv2.zip
 
+# Get gke-gcloud-auth-plugin (reqd for kubectl 1.26+)

```

Comment:
Can we remove the commented lines please :)

Reply:
Done